.circleci/config.yml

```diff
@@ -17,6 +17,15 @@ defaults:
         [ -n "$COVERAGE" -a "$CIRCLE_BRANCH" != release -a -z "$CIRCLE_TAG" ] && CMAKE_OPTIONS="$CMAKE_OPTIONS -DCOVERAGE=ON"
         cmake .. -DCMAKE_BUILD_TYPE=Release $CMAKE_OPTIONS
         make -j4
+  - run_build_ossfuzz: &run_build_ossfuzz
+      name: Build_ossfuzz
+      command: |
+        mkdir -p build
+        cd build
+        rm -rf *
```

Comment:
Does this serve any function?

Reply:
I wasn't sure if the CI environment caches contents of build dir or not, so I added the `rm -rf *` in case it does.

Reply:
No, it does not, you always have a fresh copy.